src/components/Header/index.tsx

```diff
@@ -147,18 +155,38 @@ const Header = () => {
                   {t('landing:getAccess')}
                 </UswdsReactLink>
               )}
-              <Link
-                className={classnames(
-                  'mint-header__nav-link  radius-md padding-y-105 padding-x-2',
-                  {
-                    'text-white border': isLanding,
-                    'text-base-darker': !isLanding
+              {hasSession ? (
```

Comment:
Could you add a quick comment/comments clarifying the purpose of this code?

I understand it to be: If a user has a session, the `Sign In` button should effectively use `oktaAuth.signInWithRedirect()` to skip straight to an authenticated page (`/pre-decisional-notice`), rather than just linking to the `/sign-in` route.